README.md

```diff
@@ -0,0 +1,27 @@
+# social-media-client
+
+#project is configured to run Prettier and ESLint, with pre-commit hooks.
+
+#Install prettier
```

Comment:
This is not important to passing the CA, but I recommend doing some practice with markdown as it will be more important later